test/units/modules/network/nxos/test_nxos_interface.py

```diff
@@ -89,3 +93,49 @@ def test_nxos_interface_speed_idempotence(self):
         set_module_args(dict(interface='Ethernet2/1', speed='1000'))
         result = self.execute_module(changed=False)
         self.assertEqual(result['commands'], [])
+
+    def test_nxos_interface_bfd_1(self):
+        # enable / disable part 1
+        set_module_args(dict(
+            interface='Ethernet2/1',
```

Comment:
Does this module support whitespace? `Ethernet 2/1`.  If not, should it?

Reply:
It does.